front-packages/akeneo-design-system/src/components/Placeholder/Placeholder.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {ReactElement} from 'react';
+import styled from 'styled-components';
+import {getColor, getFontSize} from '../../theme';
+import {Override} from '../../shared';
+import {IllustrationProps} from '../../illustrations/IllustrationProps';
+
+const CenteredHelperContainer = styled.div`
+  text-align: center;
+  & > * {
+    display: block;
+    margin: auto;
+  }
```

Comment:
For me you can replace it by :
```suggestion
display: flex;
flex-direction: column;
```

We try to avoid having css on children like this

Reply:
This will not work if children is "yo <span>toto</span> yo"